ctapipe/io/astropy_helpers.py

```diff
@@ -62,7 +62,7 @@ def h5_table_to_astropy(h5file, path) -> QTable:
             # need to convert to str() here so they are python strings, not
             # numpy strings
             value = table.attrs[attr]
-            other_attrs[attr] = str(value) if isinstance(value, np.str) else value
+            other_attrs[attr] = str(value) if isinstance(value, np.str_) else value
```

Comment:
```
>>> issubclass(np.str_, np.str)
True
```
So using `np.str` is probably more general.

Reply:
yes, I had that originally (see the commit message) and it worked, but what is the point of converting a str → str if it is not a numpy str_?  Doesn't hurt, but doesn't do anything either.

Reply:
unless we expect other string types, perhaps

Reply:
Ah, sorry, didn't realize that `np.str` is the python string. Sorry. That's confusing.